### ISSUE TITLE
Brown Magic not working

### ISSUE THREAD

**Issue by taercy** [xJon/The-1.12.2-Pack#567]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
** yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Issue description**
I cannot use the Brown Magic feature of PlusTiC.

It is initially unbound, and when I bind the keys, they seem to do nothing. I cannot find anything in the PlusTiC config file that would disable this feature.

**Comment by xJon** [comment]
When you try to bind to a block, are you also looking at it?

**Comment by kynexis** [comment]
It seems like the OP has not replied to you and I came across this while searching for the same issue. Binding the block seems to work, the command line comes up in the bottom left saying the position has been set but you can't "activate" brown magic to teleport to that position. I tried with multiple different hotkeys and tools etc. It just seems to not do anything.

**Comment by xJon** [comment]
You need to map the `Set Portal` & `Activate Brown Magic` buttons, then use them in that order whilst holding the appropriate tool.

**Comment by xJon** [comment]
Closed due to inactivity.
Will reopen if requested.

**Comment by Alkiery** [comment]
I am having same issue,

Are you using a legitimate launcher and Minecraft account:
yes
Are you using a computer with 8GB or more of RAM?
yes.
Are you using Java 8, 64-bit?
yes.

I've even made sure that nothing else is bound to the same buttons, I've switched the button it is bound to, there is nothing on the block that I set the target to. The message comes up that I have set the target destination but when I press g nothing happens. I can't think of anything else to troubleshoot for this.
